packages/react/src/users/hooks/useUser.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  changePassword as changePasswordAction,
+  fetchUser as fetchUserAction,
+  getAuthenticationError,
+  getChangePasswordError,
+  getLoginError,
+  getLogoutError,
+  getRecoverPasswordError,
+  getRegisterError,
+  getResetPasswordError,
+  getUser,
+  getUserError,
+  isAuthenticationLoading as isAuthenticationLoadingSelector,
```

Comment:
`isAuthenticationLoading` selector is now just an alias for `isUserLoading` as they were reflecting the same thing previously. I think we can live with just `isUserLoading` and discard using the `isAuthenticationLoading` selector. What do you think?

Reply:
Yep! will remove it 👌